cypress/support/types.ts

```diff
@@ -23,7 +24,8 @@ import {
 
 export type ItemForTest = DiscriminatedItem & {
   categories?: ItemCategory[];
-  thumbnails?: string;
+  // TODO: INCORRECT! Fix in coming
```

Comment:
Remove this when the backend is merged 😆

Reply:
Actually I will need to merge this before your packed thumbnails, so if it's okay for you I will let that and you will rebase and remove this^^

Reply:
okay no worries!